src/app/api/v1/delegates/[addressOrENSName]/route.ts

```diff
@@ -0,0 +1,13 @@
+import { NextResponse, type NextRequest } from "next/server";
+import { fetchDelegate } from "@/app/api/common/delegates/getDelegates";
+
+export async function GET(request: NextRequest) {
+    try {
+        const addressOrENSName = request.nextUrl.pathname.split('/')[4];
+        const delegate = await fetchDelegate(addressOrENSName);
```

Comment:
@Flip-Liquid here we are importing fetchDelegate which is caching getDelegate, however, since we will use fetch on the frontend it will already cache the result of this GET request by default, should I leave it as it is, or expose getDelegate without cache just for this API route?

Reply:
The /api/v1 routes are going to be authenticated, and primarily intended for use externally. I wouldn't recommend this pattern, since we'd basically need to embed an API Key on the client to run this fetch. 

Can we make the hover a server component and pre fetch the info for the VoterHoverCards being displayed while on the server? I feel like that might be more performant as well.